ibtk/include/ibtk/GeneralSolver.h

```diff
@@ -21,6 +21,8 @@
 #include "tbox/DescribedClass.h"
 #include "tbox/Pointer.h"
 
+#include <HYPRE_utilities.h>
```

Comment:
Let's not chain ourselves to the hypre interface for setting the number of iterations.

Hopefully we never need to avail ourselves of 64-bit integers to store the number of iterations used in a hypre solver. 😀